draft-ietf-tls-tlsflags.xml

```diff
@@ -129,28 +130,27 @@
                 the first octet as the flags field is transmitted.</t>
         </section>
         <section anchor="rules" title="Rules for The Flags Extension">
+            <t> Any TLS implementation that intends to propose or indicate support for a flag extension SHALL send this
+                extension with the relevant bits set.  It MUST NOT send this extension empty -- with a length of zero.</t>
+            <t> This specification does not require every flag extension to be acknowledged. Acknowledging a flag extension
+                is usually needed to inform the peer proposing the extension that the other side understands and supports
+                the extension, but some extensions do not require this acknowledgement. For example, the extension in
+                <xref target="I-D.kampanakis-tls-scas-latest" /> allows the peer to send an abbreviated certificate chain.
+                The peer need not acknowledge that it supports sending an abbreviated chain. It's enough that it sends 
+                the chain abbreviated. It is up to the document defining a particular flag extension to specify whether that
+                extension requires an acknowledgement or not.</t>
+            <t> Assuming that an acknowledgement is required, it SHOULD come at the earliest possible opportunity. A
```

Comment:
```suggestion
            <t>A
```

Earliest possible is probably not right.

1. SH is earlier than EE, but EE is usually the better choice.
2. CT is later than EE, but if the extension relates to the handling of a certificate, that is the right place.
3. HRR is earlier than anything, but unless the flag is necessary for the client to generate a new CH, HRR is the wrong place.